radixdlt-core/radixdlt/src/main/java/com/radixdlt/api/core/api.yaml

```diff
@@ -1017,11 +1044,6 @@ tags:
       the validators performing consensus. As a consequence of this, it is _not_ guaranteed to be increasing, and it
```

Comment:
I've realised I'm missing a closing bracket in the `post_transaction_state_accumulator` line above.